src/components/Sprints/SelectedSprintComonent.js

```diff
@@ -0,0 +1,162 @@
+import React, { useEffect, useReducer } from 'react';
+import {
+  Select,
+  Container,
+  Typography,
+  MenuItem,
+  FormControl,
+  InputLabel,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemSecondaryAction,
+} from '@material-ui/core';
+import RemoveIcon from '@material-ui/icons/Remove';
+import { makeStyles } from '@material-ui/core/styles';
+import '../../App.css';
+import * as dbUtils from '../../dbUtils';
+
+const useStyles = makeStyles({
+  formControl: {
+    minWidth: '100%',
+  },
+  inputLabel: {
+    color: '#bbb',
+  },
+  userInput: {
+    color: 'white',
+    width: '100%',
+  },
+});
+
+const SprintSelectionComponent = () => {
+  const classes = useStyles();
+
+  const initialState = {
+    sprint: [],
+    project: {},
+    MenuSelection: '',
+  };
+  const [state, setState] = useReducer(
+    (state, newState) => ({ ...state, ...newState }),
+    initialState
+  );
+
+  useEffect(() => {
+    loadSessionStorage();
+  }, []);
+
+  const loadSessionStorage = () => {
+    setState({
+      sprint: JSON.parse(sessionStorage.getItem('sprint')),
+      project: JSON.parse(sessionStorage.getItem('project')),
+    });
```

Comment:
does this safely handle if there isn't one in memory? if getItem returns null, does .parse not throw an error?